README.md

```diff
@@ -221,9 +224,12 @@ $ tree /opt/beman.exemplar
 │       └── exemplar
 │           └── identity.hpp
 └── lib
+    ├── cmake
+    │   ├── beman.exemplarConfig.cmake
+    │   └── beman.exemplarConfigVersion.cmake
```

Comment:
```suggestion
    │   ├── beman.exemplar-config.cmake
    │   └── beman.exemplar-config-version.cmake
```

Also, what happened to the file referenced by this bit in the config file?
`include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")`